src/sonic-yang-models/yang-models/sonic-fabric-port.yang

```diff
@@ -0,0 +1,59 @@
+module sonic-fabric-port{
+
+    yang-version 1.1;
+
+    namespace "http://github.com/sonic-net/sonic-fabric-port";
+    prefix fabric-port;
+
+    description "FABRIC_PORT yang Module for SONiC OS";
+
+    revision 2023-03-14 {
+        description "First Revision";
+    }
+
+    container sonic-fabric-port {
+
+        container FABRIC_PORT {
+
+            description "FABRIC_PORT part of config_db.json";
+
+            list FABRIC_PORT_LIST {
+
+                key "name";
+
+                leaf name {
+                    type string {
+                        length 1..128;
+                    }
+                }
+
+                leaf isolateStatus {
+                    description "Isolation status of a fabric port";
+                    type string {
```

Comment:
Why not use Boolean?

Reply:
Good suggestion. I will revise it to use `type stypes:boolean_type` where stypes is sonic-types.

Reply:
Done. See latest commit. I'm waiting for the pipeline build to complete to check if any further revisions are needed.